src/Grid/RemoteDataSource.js

```diff
@@ -81,8 +81,13 @@ class RemoteDataSource {
       
         return obj;
       });
-
-      this.dataStream.onNext({ Payload: rows, Aggregate: response.data.AggregationPayload });
+  
+      this.dataStream.onNext({ 
+        Payload: rows, 
```

Comment:
We should change this to camelcase the property name on the `dataStream `next  object